docs/pages/mintlist/[mintlistAddress].tsx

```diff
@@ -284,6 +304,90 @@ function useMintlist({
   return { data, error };
 }
 
+function MintButton({
+  mintlist,
+  network,
+}: {
+  mintlist: NftokenTypes.Mintlist;
+  network: Network;
+}) {
+  const [minting, setMinting] = useState(false);
```

Comment:
(This isn't important.) I like to use `useBoolean` since it has a nicer API for setting true / false.